services/ui-src/src/components/app/App.tsx

```diff
@@ -34,7 +34,7 @@ export const App = () => {
   const authenticatedRoutes = (
     <>
       {user && (
-        <Flex data-testid="app-container" sx={sx.appLayout}>
+        <Flex sx={sx.appLayout}>
```

Comment:
is this the `data-testid` for App.test.tsx to pass?

Reply:
the data-testid should be attached to the container element so i removed it from flex -- i had it there temporarily to get the test to pass in the last PR